spec/javascripts/support/dom.js

```diff
@@ -27,7 +27,6 @@ export function createDOM() {
           : Promise.reject(new Error('Failed to load'));
       }
     })(),
-    runScripts: 'dangerously',
```

Comment:
I noticed that `Object.getOwnPropertyNames(window)` would return a different result with this assigned, and was missing many properties (e.g. `window.Element`). After tracking down why we needed this in the first place, I came up with the substitute implementation below, which is also similar to [what we do for images](https://github.com/18F/identity-idp/blob/ef00bf1ba02ee5ff5faad6b725e2e07ed026f349/spec/javascripts/spec_helper.js#L28-L32).